app.js

```diff
@@ -22,7 +22,14 @@ const app = express();
 initPassport();
 
 app.enable('trust proxy');
-app.use(cors());
+
+const corsOption = {
```

Comment:
Cors does not work without those options specified?

Reply:
Cors work normal. I thought it will be nice to configure it for the extra http verbs.

Reply:
Which are the extra verbs? And are you using any extra verb that did not work before now?

Reply:
I later did not use them. I have undo the changes